### DISCUSSION TITLE
Wazuh agent can't connect to manager. (Can't telnet to 1514 even with firwalld disabled)

### DISCUSSION THREAD

**Discussion by jcastillo725** [Security-Onion-Solutions/securityonion#3787] [Unsupported Versions]
I am trying to connect wazuh client to the manager but cannot. I found out that I can't telnet to the management node with port 1514 even though I did so-allow on the IP range and even on the specific IP. None of the other previous solutions here have worked. 

I'm currently testing it for a VM lab environment. I have an opnsense and verified that the connection is allowed.

**Comment by B3DTech** [comment]
So - I needed to remove the client.keys file from my agent and re-register the agent to SO.

And even though Wazuh documentation says to update the port to 1514 TCP in the config, you still need to use 1514 UDP in the agent.

Once I did both of these and restart the agent, it was communicating successfully.

**Reply by jcastillo725** [reply]
Thank you, will try this and give an update if this works for me.
